### ISSUE TITLE
[a11y]: Reporting Periods - accessibility findings

### ISSUE THREAD

**Issue by lsr-explore** [usdigitalresponse/usdr-gost#3897]
### Why is this issue important?

This ticket contains multiple accessibility issues in arpa_reporter.  As support for Arpa reporter is winding down we collected issues in one page.  

Note:  These were identified during a developer accessibility review to double check the accessibility of the elements on the page.   To fully insure the site is accessible to users, more comprehensive testing by an accessibility subject matter expert should be performed.

Testing approach
All testing was done against [staging](https://staging.grants.usdr.dev/arpa_reporter) and local dev.
1) Run [axe DevTools chrome extension ](https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US)against the page.   This checks contrast 
2) Run through the page with the keyboard - I used Microsoft's [Accessibility Insights Chrome Extension](https://chromewebstore.google.com/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni) to track keyboard focus
3) Zoom the page to 200x and 400x to check reflow
4) Run through the page using a screen reader - mac voice over
5) Validate the HTML by pasting parts of the html into [W3C Markup Validation Service](https://validator.w3.org/#validate_by_input)

Testing gaps
1) Windows screen readers - JAWS / NVDA
2) Mobile web devices - the arpa reporter was not designed to be responsive
3) Real user workflow steps.   

----

# Reporting Periods

## Issue 1 - table buttons/links need aria-labels for additional context.

* Upload Template
* Certify Reporting Period
* Edit

![Image](https://github.com/user-attachments/assets/e7a42564-0788-443d-8835-c7e50539c61a)

## Issue 2 - Table Header text should not be empty

![Image](https://github.com/user-attachments/assets/3a1df8b5-6675-4caa-ac93-127877054fba)

### Current State

See individual issues above

### Expected State

See individual issues above

### Implementation Plan

tbd

### Relevant Code Snippets

```shell
tbd
```
